cli/tests/integrations/common.py

```diff
@@ -547,14 +548,22 @@ def ssh_output(cmd):
     :rtype: (str, str)
```

Comment:
Change the documentation to reflect that it returns a tuple with 3 elements: str, str, int.